packages/mui-material/src/Grid/Grid.js

```diff
@@ -529,6 +523,65 @@ Grid.propTypes /* remove-proptypes */ = {
     PropTypes.arrayOf(PropTypes.oneOf(['column-reverse', 'column', 'row-reverse', 'row'])),
     PropTypes.object,
   ]),
+  /**
+   * @ignore
+   */
+  flexWrap: PropTypes.oneOfType([
```

Comment:
The Grid component already supports the `flexWrap` prop, so we shouldn't have to add it here.